index.html

```diff
@@ -157,87 +157,87 @@ <h2>Stages</h2>
 
-<p>At stages 2 and later, the semantics, API and syntax of an addition must be described as edits to the latest published ECMAScript standard, using the same language and conventions. The quality of the spec text expected at each stage is described above.
+<p>At Stage 2 and later, the semantics, API, and syntax of an addition must be described as edits to the latest published ECMAScript standard, using the same language and conventions. The quality of the spec text expected at each stage is described above.
 
 <h2>Reviewers</h2>
 
 <p>Anyone can be a reviewer and submit feedback on an in-process addition. The committee should identify designated reviewers for acceptance during Stage 2. These reviewers must give their sign-off before a proposal enters Stage 2.7. Designated reviewers should not be authors of the spec text for the addition and should have expertise applicable to the subject matter. Designated reviewers must be chosen by the committee, not by the proposal's champion.
 
-<p>When reviewers are designated, a target meeting for Stage 2.7 should be identified. Initial reviewer feedback should be given to the champions two weeks before that meeting to allow for a back-and-forth ahead of the meeting. The target Stage 2.7 meeting may be delayed by a champion outside of the meeting at a later time if it is not ready.
+<p>When reviewers are designated, a target meeting for Stage 2.7 should be identified. Initial reviewer feedback should be given to the champions at least two weeks before the meeting to allow time for discussion and revisions. The target Stage 2.7 meeting may be postponed by a champion if further refinement is necessary.
 
 <h2>Calls for implementation and feedback</h2>
 
-<p>When an addition is accepted at stage 3, the committee is signifying that it believes design work is complete and further refinement will require implementation experience, significant usage, and external feedback.
+<p>When an addition is accepted at Stage 3, the committee is signifying that design work is complete and further refinement will require implementation experience, significant usage, and external feedback.
 
 <h2>Tips for achieving consensus</h2>
 
-<p>During the discussion of a proposal any aspect may be discussed. Consensus is given as an indicator of the current stage. Delegates should openly give feedback on proposals, and especially for a proposal for stage advancement where the concern is relevant to the stage. Delegates should raise their concerns early and asynchronously, in order to help the champion resolve any issues.
+<p>During the discussion of a proposal, any aspect may be addressed. Delegates are encouraged to provide feedback on proposals, especially on stage advancement proposals where concerns are relevant to that stage. Feedback and concerns should be raised early and asynchronously to help the champion resolve any issues efficiently. Consensus determines whether a proposal is ready to advance to the next stage.
```

Comment:
I don't think "stage advancement proposal" is a coherent concept.
```suggestion
<p>During the discussion of a proposal, any aspect may be addressed. Delegates are encouraged to provide feedback on proposals, especially when it is relevant to stage advancement. Feedback and concerns should be raised early and asynchronously to help the champion resolve any issues efficiently. Consensus determines whether a proposal is ready to advance to the next stage.
```

Reply:
what it is trying to convey is that feedback is especially important when the proposal being discussed is seeking stage advancement at the time of the discussion.  in other words a topic like `proposal x for stage 2` vs `proposal x stage 1 update`

Reply:
I understand the intent, I just don't think the potentially confusing phrase should remain.